eukaryote/constraints/semantics/sentence_encoders/universal_sentence_encoder/universal_sentence_encoder.py

```diff
@@ -28,12 +27,8 @@ def __init__(self, threshold=0.8, large=False, metric="angular", **kwargs):
     def encode(self, sentences):
         if not self.model:
             self.model = hub.load(self._tfhub_url)
-        encoding = self.model(sentences)
-
-        if isinstance(encoding, dict):
```

Comment:
worried that removing this check might break something we're not aware of, flag for caution